lib/tasks/dev.rake

```diff
@@ -0,0 +1,77 @@
+
+task sample_data: :environment do
+  p "creating sample data"
+  starting = Time.now
+
+  Member.delete_all
+  Contact.delete_all
+  Group.delete_all
+  User.delete_all
+
+  5.times do
+    number = rand.to_s[2..11] 
+    name = Faker::Name.first_name
+    User.create(
+      email: "#{name}@example.com",
+      password: "password",
+      username: name.downcase,
+      phone_number: "1#{number}",
+    )
+  end
+
+  number = rand.to_s[2..11]
+  User.create(
+    email: "alice@example.com",
+    password: "password",
+    username: "alice",
+    phone_number: "1#{number}",
+  )
+
+  users = User.all
+
+  users.each do |u|
+
+    30.times do
+      p_number = rand.to_s[2..11]
+      w_number =  rand.to_s[2..11]
+      o_number =  rand.to_s[2..11]
+      name = Faker::Name.unique.first_name
+      u.contacts.create(
+        name: name,
+        email: "#{name}@example.com",
+        personal_number: "1#{p_number}",
+        work_number: "1#{w_number}",
+        facebook_url: "facebook.com",
+        instagram_url: "instagram.com",
+        preferred_method: ["text", "text (work)", "phone call", "phone call (work)", "email", "facebook", "instagram"].sample,
```

Comment:
I see that you left a comment to make `preferred_method` an enum column.

Once you do, you could change this to

```suggestion
        preferred_method: Contact.preferred_methods.sample,
```

There are a lot of other things you can read about [here](https://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html)